apps/mobile/src/utils/localization/I18nProvider.tsx

```diff
@@ -18,6 +18,7 @@ export const supportedTranslations = {
   pl: translations.pl,
   bg: translations.bg,
   ja: translations.ja,
+  es: translations.es,
```

Comment:
Pls add translation also to `useSetRelativeDateFormatting.ts` as an import like:
`import 'dayjs/locale/es'`